packages/data-explorer/src/store/state.ts

```diff
@@ -1,6 +1,6 @@
-import ApplicationState from '@/types/ApplicationState'
+import { AppState } from '@/types/ApplicationState'
```

Comment:
why this change to AppState, ApplicationState seems clearer to me, and i do not understand the link to show/hide the settings btn